Gemfile

```diff
@@ -42,8 +40,7 @@ gem 'ace-rails-ap'
 # Serializers
 gem 'rdf-n3'
 gem 'jbuilder'
-gem 'yajl-ruby'
```

Comment:
Replaced with OJ as its what we're using elsewhere. Faster benchmarks too, but probably insignificant in the grand scheme of things.